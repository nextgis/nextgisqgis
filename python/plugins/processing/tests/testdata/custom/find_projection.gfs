<GMLFeatureClassList>
  <GMLFeatureClass>
    <Name>find_projection</Name>
    <ElementPath>find_projection</ElementPath>
    <!--POINT-->
    <GeometryType>1</GeometryType>
    <SRSName>EPSG:28356</SRSName>
    <DatasetSpecificInfo>
      <FeatureCount>4</FeatureCount>
      <ExtentXMin>326064.87345</ExtentXMin>
      <ExtentXMax>328710.85361</ExtentXMax>
      <ExtentYMin>6245538.80829</ExtentYMin>
      <ExtentYMax>6247808.84283</ExtentYMax>
    </DatasetSpecificInfo>
  </GMLFeatureClass>
</GMLFeatureClassList>
