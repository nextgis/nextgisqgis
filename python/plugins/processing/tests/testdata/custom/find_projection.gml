<?xml version="1.0" encoding="utf-8" ?>
<ogr:FeatureCollection
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation=""
     xmlns:ogr="http://ogr.maptools.org/"
     xmlns:gml="http://www.opengis.net/gml">
  <gml:boundedBy>
    <gml:Box>
      <gml:coord><gml:X>326064.8734538725</gml:X><gml:Y>6245538.808291084</gml:Y></gml:coord>
      <gml:coord><gml:X>328710.8536064086</gml:X><gml:Y>6247808.842832778</gml:Y></gml:coord>
    </gml:Box>
  </gml:boundedBy>
                                                                                                
  <gml:featureMember>
    <ogr:find_projection fid="find_projection.0">
      <ogr:geometryProperty><gml:Point srsName="EPSG:28356"><gml:coordinates>326597.10210384,6247528.48897066</gml:coordinates></gml:Point></ogr:geometryProperty>
    </ogr:find_projection>
  </gml:featureMember>
  <gml:featureMember>
    <ogr:find_projection fid="find_projection.1">
      <ogr:geometryProperty><gml:Point srsName="EPSG:28356"><gml:coordinates>327449.712845641,6246290.065952</gml:coordinates></gml:Point></ogr:geometryProperty>
    </ogr:find_projection>
  </gml:featureMember>
  <gml:featureMember>
    <ogr:find_projection fid="find_projection.2">
      <ogr:geometryProperty><gml:Point srsName="EPSG:28356"><gml:coordinates>328710.853606409,6247808.84283278</gml:coordinates></gml:Point></ogr:geometryProperty>
    </ogr:find_projection>
  </gml:featureMember>
  <gml:featureMember>
    <ogr:find_projection fid="find_projection.3">
      <ogr:geometryProperty><gml:Point srsName="EPSG:28356"><gml:coordinates>326064.873453873,6245538.80829108</gml:coordinates></gml:Point></ogr:geometryProperty>
    </ogr:find_projection>
  </gml:featureMember>
</ogr:FeatureCollection>
