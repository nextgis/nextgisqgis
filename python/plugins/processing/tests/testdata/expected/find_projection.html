<html><head>
<meta http-equiv="Content-Type" content="text/html;                     charset=utf-8" /></head><body>
<p>EPSG:20256</p>
<p>EPSG:20356</p>
<p>EPSG:28356</p>
<p>EPSG:32356</p>
<p>EPSG:32556</p>
<p>EPSG:32756</p>
<p>IGNF:UTM56SW84</p>
<p>EPSG:5552</p>
</body></html>
